apps/chat/src/store/conversations/conversations.epics.ts

```diff
@@ -199,37 +196,39 @@ const getSelectedConversationsEpic: AppEpic = (action$, state$) =>
                   }),
                 ),
               ),
+            ).pipe(
+              map((conversations) => {
+                const validConversations = conversations.filter(
+                  Boolean,
+                ) as Conversation[];
+
+                return validConversations
+                  .filter(
+                    ({ messages }) =>
+                      !messages.filter((m) => m.role !== Role.System).length,
+                  )
+                  .map(regenerateConversationId);
+              }),
             ),
             selectedIds: of(selectedIds),
           });
         }),
         map(({ selectedConversations, selectedIds }) => {
-          const conversations = selectedConversations
-            .filter(
-              (conv) =>
-                !!conv &&
-                (!payload?.createNew ||
-                  !conv.messages.filter((m) => m.role !== Role.System).length),
```

Comment:
without this condition now (w/o #2584) new conversation will be created each time if even selected one already is new